.github/workflows/ci.yml

```diff
@@ -59,4 +59,12 @@ jobs:
 
     - name: Run lint check for feature-management-applicationinsights-browser
       run: npm run lint
-      working-directory: sdk/feature-management-applicationinsights-browser
\ No newline at end of file
+      working-directory: sdk/feature-management-applicationinsights-browser
+
+    - name: Build feature-management-applicationinsights-node
+      run: npm run build
+      working-directory: sdk/feature-management-applicationinsights-node
+
+    - name: Run lint check for feature-management-applicationinsights-node

```

Comment:
nit: should we lint before build? Just so it fails faster

Reply:
Sounds good. Will create another pr to do this